openml/setups/functions.py

```diff
@@ -4,20 +4,23 @@
 import xmltodict
 
 from .setup import OpenMLSetup, OpenMLParameter
+from openml.flows import sklearn_to_flow, flow_exists
 
 
-def setup_exists(downloaded_flow, sklearn_model):
+def setup_exists(flow, model=None):
     '''
-    Checks whether a flow / hyperparameter configuration already exists on the server
+    Checks whether a hyperparameter configuration already exists on the server.
```

Comment:
What do you mean by models do not agree? You mean names don't match? Or the parameter names don't match? Or the parameter values?

Matching of parameter names is done somewhat in _parse_parameters, but I could make that more strict (I'll think about it...)